.github/teams.yml

```diff
@@ -0,0 +1,35 @@
+Admin:
```

Comment:
is this file duplicated? you have a `teams.yaml` and a `teams.yml`

Reply:
yeah.. it was duplicated..just removed